core/pubspec.yaml

```diff
@@ -42,6 +42,8 @@ dependencies:
 
   built_collection: 5.1.0
 
+  html: 0.15.0
```

Comment:
Why you not use `sanitize_html` instead of parsing and sanitize yourself?

Reply:
Since sanitize_html removes all the attributes in the tags that I need to keep. (Ex: style)

Reply:
No way to modified it?. How about fork it and make a change. We should to avoid to re-invent the cycle as much as possible